LFGBulletinBoard/Dungeons.lua

```diff
@@ -111,6 +111,7 @@ function GBB.GetDungeonNames()
 		["BREW"] =  "Brewfest - Coren Direbrew",
 		["HOLLOW"] =  "Hallow's End - Headless Horseman",
     	["TRAVEL"] = "Travel services - Summons/Portals",
+		["DFC"] = "Demon Fall Canyon",
```

Comment:
For future reference, adding into `DefaultEnGB` is not required if the dungeon is defined `dungeons/classic.lua`. 

For this reason, the `assert` checks in `dungeons/classic.lua` were made to error on addon-load if no "name" info could be found from the associated `LFGDungeonID`/`GroupFinderActivityID` and no fallback is provided in the `infoOverrides` table. 

The goal was to deprecate the `DefaultEnGB` table in `GBB.GetDungeonNames` eventually, so that way all dungeon related instantiation is done inside of `dungeons/{expansion}.lua`